v63003/u_inref/gtm8587.csh

```diff
@@ -16,7 +16,12 @@
 set key='$KEY'
 set device='$DEVICE'
 setenv gtm_chset "UTF-8"
+echo "# Triggering an IO Permissions Error, expecting a message in $device (would be zero in previous versions)"
+$ydb_dist/mumps -run dollardevice^gtm8587
+echo ""
 echo "# RUNNING TEST FOR $key\rIF YOU SEE THIS THE TEST FAILED">>&temp1.txt
+echo "# Ending a read with a new line control character, expecting $key to reflect this (would be null in previous versions)"
 $ydb_dist/mumps -run dollarkey^gtm8587 temp1.txt
-echo "# RUNNING TEST FOR $device"
-$ydb_dist/mumps -run dollardevice^gtm8587
+echo ""
+echo "# Ending a read with an EOF, expecting $key to be null"
```

Comment:
The echo says we are ending a read with an EOF but we are not passing any parameter to dollarkey^gtm8587. So what file does it open? Is it really testing read EOF.

Reply:
It is not opening a file, when a READ encounters an empty file it terminates as an EOF (I believe the website mentions this). I provided $device to confirm that the read was terminated by this, but I can change it so it reads an actual file and gets terminated by an eof if you would prefer that

Reply:
Agree $DEVICE is confirmation enough. Thanks. No more changes needed.